src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+    reset() {
+      obj.result = 0;
+
+      return this;
+    },
+    add(num) {
```

Comment:
```suggestion
    add(num = 0) {
```
let's add default values for operations parameters